universal-application-tool-0.0.1/app/services/question/QuestionDefinitionBuilder.java

```diff
@@ -0,0 +1,91 @@
+package services.question;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Locale;
+import java.util.Optional;
+
+public class QuestionDefinitionBuilder {
```

Comment:
Thought (non-blocking): this could also be `QuestionDefinition.Builder` in the same file like how autovalue does it, I'm fine either way though

Reply:
I would have preferred that as well but I couldn't do that because that'd create a circular dependency.

QuestionDefinition doesn't know about AddressQuestionDefinition.